cfme/common/vm.py

```diff
@@ -285,7 +285,7 @@ def find_quadicon(
             elif self.is_vm:
                 self.provider.load_all_provider_vms()
             else:
-                self.provider.load_all_provider_templates()
+                self.provider.load_all_provider_images()
```

Comment:
Why changing this? On line 521 in `cfme/common/providers.py`, it is just an alias for `load_all_provider_templates`.